testing/trino-tests/src/test/java/io/trino/tests/TestMockConnector.java

```diff
@@ -105,7 +112,7 @@ public void testRenameSchema()
     @Test
     public void testCreateMaterializedView()
     {
-        assertUpdate("CREATE MATERIALIZED VIEW mock.default.materialized_view AS SELECT * FROM tpch.tiny.nation");
+        assertUpdate("CREATE MATERIALIZED VIEW mock.default.materialized_view WITH (refresh_interval = '1h') AS SELECT * FROM tpch.tiny.nation");
```

Comment:
what does it test?

Reply:
It tests whether `MockConnector.getMaterializedViewProperties` and `MockConnectorFactory.withGetMaterializedViewProperties` work as expected.